content/docs/rfcs/13/README.md

```diff
@@ -105,103 +112,144 @@ message HistoryRPC {
 
 ### Index
 
-To perform pagination, each `WakuMessage` stored at a node running the `13/WAKU2-STORE` protocol is associated with a unique `Index` that encapsulates the following parts. 
+To perform pagination, 
+each `WakuMessage` stored at a node running the `13/WAKU2-STORE` protocol is associated with a unique `Index` that encapsulates the following parts. 
 - `digest`:  a sequence of bytes representing the SHA256 hash of a `WakuMessage`.
   The hash is computed over the concatenation of `contentTopic` and `payload` fields of a `WakuMessage` (see [14/WAKU2-MESSAGE](/spec/14)).
-- `receiverTime`: the UNIX time in nanoseconds at which the waku message is received by the receiving node.
-- `senderTime`: the UNIX time in nanoseconds at which the waku message is generated by its sender.
-- `pubsubTopic`: the pubsub topic on which the waku message is received
+- `receiverTime`: the UNIX time in nanoseconds at which the `WakuMessage` is received by the receiving node.
+- `senderTime`: the UNIX time in nanoseconds at which the `WakuMessage` is generated by its sender.
+- `pubsubTopic`: the pubsub topic on which the `WakuMessage` is received.
 
 ### PagingInfo
 
 `PagingInfo` holds the information required for pagination. It consists of the following components.
-- `pageSize`: A positive integer indicating the number of queried `WakuMessage`s in a `HistoryQuery` (or retrieved  `WakuMessage`s in a `HistoryResponse`).
+- `pageSize`: A positive integer indicating the number of queried `WakuMessage` in a `HistoryQuery` (or retrieved  `WakuMessage` in a `HistoryResponse`).
```

Comment:
I think is interesting to keep the notion of possible multiple messages being retrieved, which is the usual case.